tests/oldcheck/check_test.go

```diff
@@ -54,50 +47,21 @@ func testCheck(t *testing.T, engine string) {
 	data, err := os.ReadFile("tests.yaml")
 	require.NoError(t, err)
 
-	var tests checkTests
-	err = yaml.Unmarshal(data, &tests)
+	var tt checkTests
```

Comment:
`tt` isn't super clear. Mind if we leave this as `tests` or `testCases`?

Reply:
The package name, for better or worse, is tests, so can't use that. I'll go with testCases.

Reply:
oh, lol. Yeah, `testCases` is a little better 👍 . Thanks, I know that was a bit of a nit pick..